From 5b890ec5160e3080d0f3a55f94cf3f95fc6fd611 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Micha=C5=82=20G=C3=B3rny?= <mgorny@gentoo.org>
Date: Mon, 16 May 2022 10:32:20 +0200
Subject: [PATCH 19/22] Fix test_nntplib backports for 3.8 code

---
 Lib/test/test_nntplib.py | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/Lib/test/test_nntplib.py b/Lib/test/test_nntplib.py
index d27e4abae8..b0f462baa7 100644
--- a/Lib/test/test_nntplib.py
+++ b/Lib/test/test_nntplib.py
@@ -272,7 +272,7 @@ class NetworkedNNTPTestsMixin:
             return True
 
         kwargs = dict(
-            timeout=support.INTERNET_TIMEOUT,
+            timeout=TIMEOUT,
             usenetrc=False
         )
         if self.ssl_context is not None:
@@ -316,12 +316,12 @@ class NetworkedNNTPTests(NetworkedNNTPTestsMixin, unittest.TestCase):
     def setUpClass(cls):
         support.requires("network")
         kwargs = dict(
-            timeout=support.INTERNET_TIMEOUT,
+            timeout=TIMEOUT,
             usenetrc=False
         )
         if cls.ssl_context is not None:
             kwargs["ssl_context"] = cls.ssl_context
-        with socket_helper.transient_internet(cls.NNTP_HOST):
+        with support.transient_internet(cls.NNTP_HOST):
             try:
                 cls.server = cls.NNTP_CLASS(cls.NNTP_HOST, **kwargs)
             except SSLError as ssl_err:
-- 
2.35.1

